cwms-data-api/src/main/java/cwms/cda/api/LocationController.java

```diff
@@ -296,8 +307,15 @@ public void create(@NotNull Context ctx) {
             String formatHeader = ctx.req.getContentType();
             ContentType contentType = Formats.parseHeader(formatHeader, Location.class);
             Location locationFromBody = Formats.parseContent(contentType, ctx.body(), Location.class);
-            locationsDao.storeLocation(locationFromBody);
-            ctx.status(HttpServletResponse.SC_OK).json("Created Location");
+            boolean failIfExists = ctx.queryParamAsClass(FAIL_IF_EXISTS, Boolean.class).getOrDefault(true);
+            locationsDao.storeLocation(locationFromBody, failIfExists);
+            ctx.status(HttpServletResponse.SC_CREATED).json("Created Location");
+        } catch (AlreadyExists e) {
```

Comment:
This controller has the error handling within as it's one of the first and was created before we established the error handling within the ApiServlet exception definition.

New code should follow the new scheme for error handling.